.github/workflows/ci.yml

```diff
@@ -134,7 +134,7 @@ jobs:
         uses: SimenB/github-actions-cpu-cores@v2
 
       - name: 🧪 Test
-        run: yarn test-ci ${{ steps.cpu-cores.outputs.count }}
+        run: yarn test-ci --minWorkers=1 --maxWorkers=${{ steps.cpu-cores.outputs.count }}
```

Comment:
Nice! Not sure why the config was split between two files probably an oversight on my part